seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/resources/kafka/fake_to_kafka_exactly_once.conf

```diff
@@ -0,0 +1,48 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+env {
+    execution.parallelism = 1
+    job.mode = "BATCH"
```

Comment:
@fcb-xiaobo  hi, can we add an e2e for Streaming

Reply:
ok,i'll add it

Reply:
I am running the example module locally, SeaTunnelApiExample, using the following configuration

env {
    execution.parallelism = 1
    job.mode = "STREAMING"
  }

source {
  Kafka {
    bootstrap.servers = "xxx:8092"
    topic = "test_topic_source"
    # The default format is json, which is optional
    format = json
    start_mode = earliest
    }
  }



transform {}


sink{
  kafka {
        format = JSON
        topic = "test01"
        bootstrap.servers = "xxx:8092"
        semantics = EXACTLY_ONCE
    }
}


I use the Kafka producer console to send messages, and then open the consumer console to consume messages
Problem 1: Messages are only consumed during startup each time
Problem 2: The seatunnel log reports an error, as shown in the figure below, but the Flink UI checkpoints are all successful
![image](https://github.com/user-attachments/assets/15d252e5-cb96-4405-9d73-391605292c9b)

Problem 3: During the startup process, after sending a new message, consumers cannot consume it anymore
I have also debugged the source code locally and did not find any specific issues. I am not sure if this is a bug or if my usage is incorrect. I would like to ask for your advice. If it is convenient for you, you can communicate directly on WeChat, which will be more convenient

Reply:
link https://github.com/apache/seatunnel/pull/7871